core/src/main/java/org/elasticsearch/search/builder/SearchSourceBuilder.java

```diff
@@ -315,7 +315,7 @@ public QueryBuilder postFilter() {
      */
     public SearchSourceBuilder from(int from) {
         if (from < 0) {
-            throw new IllegalArgumentException("[from] parameter cannot be negative");
+            throw new IllegalArgumentException("[from] parameter cannot be negative but was [\" + from + \"]");
```

Comment:
Because you escape the quotation marks this will literally print `" + from + "`. But I think that you want to print the value of from.